playbooks/manageiq-providers-openstack-test-devstack/run.yaml

```diff
@@ -11,13 +11,50 @@
     - install-devstack
     - clean-devstack-default-resources
   tasks:
+    - name: Install specifc packages and config Neutron for Ubuntu 14.04
+      shell:
+        cmd: |
+          set -ex
+          apt-get -y install software-properties-common
+          apt-add-repository ppa:brightbox/ruby-ng -y
+          echo "deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main 9.5" >> /etc/apt/sources.list.d/pgdg.list
+          curl -4 -O https://www.postgresql.org/media/keys/ACCC4CF8.asc
+          apt-key add ACCC4CF8.asc
+          apt-get -o Acquire::ForceIPv4=true update -y
+          apt-get -o Acquire::ForceIPv4=true install postgresql-9.5 -y
+          psql --version
+          apt-get install ruby2.3 ruby2.3-dev -y
+          ruby -v
+          sed -i "/flat_networks/ a flat_networks = public,public_net_0,public_net_1,public_net_2,public_net_3,public_net_4,public_net_5" /etc/neutron/plugins/ml2/ml2_conf.ini
+          killall neutron-server
+          nohup /usr/bin/python /usr/local/bin/neutron-server --config-file /etc/neutron/neutron.conf --config-file /etc/neutron/plugins/ml2/ml2_conf.ini > /dev/null 2>&1 &
+        executable: /bin/bash
+        chdir: '{{ zuul.project.src_dir }}'
+      when:
+        - global_env.OS_BRANCH == 'stable/mitaka'
+
+    - name: Install specifc packages and config Neutron for Ubuntu 16.04
+      shell:
+        cmd: |
+          set -ex
+          # Install Ruby
+          apt-get install -y build-essential zlibc zlib1g-dev ruby ruby-dev openssl libxslt-dev libxml2-dev \
+              libssl-dev libreadline6 libreadline6-dev libyaml-dev libsqlite3-dev sqlite3
+          apt-get install -y postgresql python-psycopg2 libpq-dev
```

Comment:
`apt-get install -y python-psycopg2 libpq-dev` command will be executed in line 83, so we should only install `postgresql` at here.

Reply:
thx done